org.skytemple.SkyTemple.yml

```
@@ -0,0 +1,134 @@
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: cmake-ninja
+    no-make-install: true
+    build-options:
+      cxxflags: "-DCMAKE_BUILD_TYPE=Release"
+    post-install:
+      - install -Dm755 armips /app/bin/armips
+    sources:
+      - type: archive
+        url: https://github.com/Kingcom/armips/archive/v0.11.0.tar.gz
+        sha256: c94e29dfda3bdf853590d825562b9c73a3d8e8e995555e021c6b2a6451572681
+      - type: patch
+        path: patches/armips.patch
+  # This needs to be done seperately due to some patches and the Rust compiler and Cargo packages required.
+  # Also this needs build isolation, unlike the rest.
+  - name: python3-skytemple-rust
+    buildsystem: simple
+    build-commands:
+      - . /usr/lib/sdk/rust-nightly/enable.sh && pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "./skytemple-rust-1.3.7"
+    build-options:
+      env:
+        CARGO_HOME: /run/build/python3-skytemple-rust/cargo
+    sources:
+      - cargo-sources.json
+      - type: file
+        url: https://files.pythonhosted.org/packages/27/d6/003e593296a85fd6ed616ed962795b2f87709c3eee2bca4f6d0fe55c6d00/wheel-0.37.1-py2.py3-none-any.whl
+        sha256: 4bdcd7d840138086126cd09254dc6195fb4fc6f01c050a1d7236f2630db1d22a
+      - type: file
+        url: https://files.pythonhosted.org/packages/2e/49/565350c6fab3f5a3e2c46633290117060e70e2501544cdf3bde1d1d5d0fe/range_typed_integers-1.0.1-py3-none-any.whl
+        sha256: 35d39a41642503c5c5117e26798713f081b1beece1b2afd8f1ba70c8d90f63c5
+      - type: file
+        url: https://files.pythonhosted.org/packages/6a/23/8146aad7d88f4fcb3a6218f41a60f6c2d4e3a72de72da1825dc7c8f7877c/semantic_version-2.10.0-py2.py3-none-any.whl
+        sha256: de78a3b8e0feda74cabc54aab2da702113e33ac9d9eb9d2389bcf1f58b7d9177
+      - type: file
+        url: https://files.pythonhosted.org/packages/6a/23/8146aad7d88f4fcb3a6218f41a60f6c2d4e3a72de72da1825dc7c8f7877c/semantic_version-2.10.0-py2.py3-none-any.whl
+        sha256: de78a3b8e0feda74cabc54aab2da702113e33ac9d9eb9d2389bcf1f58b7d9177
+        type: file
+        url: https://files.pythonhosted.org/packages/ed/d6/2afc375a8d55b8be879d6b4986d4f69f01115e795e36827fd3a40166028b/typing_extensions-4.3.0-py3-none-any.whl
+        sha256: 25642c956049920a5aa49edcdd6ab1e06d7e5d467fc00e0506c44ac86fbfca02
+      - type: file
+        url: https://files.pythonhosted.org/packages/d9/5f/2daccd14278b6b780ae6799f85998377c06019354982391245f4b58a927d/setuptools-65.3.0-py3-none-any.whl
+        sha256: 2e24e0bec025f035a2e72cdd1961119f557d78ad331bb00ff82efb2ab8da8e82
+      # At the moment there is a packaging issue with the source on PyPi: The macro subcrates are not contained in the source tarballs.
+      # We need to fix this upstream, but in the meantime we use the Git repo.
+      - type: git
+        url: https://github.com/SkyTemple/skytemple-rust.git
+        tag: 1.3.7
+        dest: skytemple-rust-1.3.7
```

```suggestion
        dest: skytemple-rust
```

I would just not put any version